<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_PinMap</class>
 <widget class="QWidget" name="Form_PinMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <widget class="QRadioButton" name="rb_pin7">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QRadioButton" name="rb_pin2">
      <property name="text">
       <string>2</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QRadioButton" name="rb_pin3">
      <property name="text">
       <string>3</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QRadioButton" name="rb_pin1">
      <property name="text">
       <string>1</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="rb_pin9">
      <property name="text">
       <string>9</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QRadioButton" name="rb_pin10">
      <property name="text">
       <string>10</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="rb_pin8">
      <property name="text">
       <string>8</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="rb_pin4">
      <property name="text">
       <string>4</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="rb_pin5">
      <property name="text">
       <string>5</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QRadioButton" name="rb_pin6">
      <property name="text">
       <string>6</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
